### DISCUSSION TITLE
Support by default for new (in Clojure 1.11) core fns and namespaces?

### DISCUSSION THREAD

**Discussion by pmonks** [borkdude/sci#953] [Q&A]
### Problem statement
I use `sci` (with some scaffolding) as a sandbox where beginners can easily learn Clojure basics.  This sometimes verges into math, where new (in Clojure 1.11) fns and namespaces are particularly useful.

Examples include (but are not limited to):
* [`clojure.core/parse-long`](https://clojuredocs.org/clojure.core/parse-long)
* [`clojure.core/parse-double`](https://clojuredocs.org/clojure.core/parse-double)
* the [`clojure.math` namespace](https://clojuredocs.org/clojure.math)

### Potential solutions (by no means exhaustive)
I would like these core fns and namespaces to be available to `sci`-executed code by default, just the same as other, older core fns and namespaces are.

### Potential alternatives (by no means exhaustive)
In my own scaffolding code I could manually expose these new core fns and namespaces to the `sci` environment, as described in the `sci` docs [\[1\]](https://github.com/babashka/sci/tree/master?tab=readme-ov-file#vars) [\[2\]](https://github.com/babashka/sci/tree/master?tab=readme-ov-file#copy-a-namespace)

### Additional context
The "scaffolding" I've written happens to be an interactive chat bot, just in case that context is relevant for this issue.

**Comment by borkdude** [comment]
you can use this re-usable configuration from sci.configs:

https://github.com/babashka/sci.configs/blob/main/src/sci/configs/clojure_1_11.cljc

See the README on how to use those in general

**Comment by borkdude** [comment]
As for clojure.math, you can just use `sci/copy-ns` for that

**Comment by pmonks** [comment]
@borkdude apologies if my initial description wasn't clear - what I was requesting was that the new (in Clojure 1.11) fns and `clojure.math` namespace would be available in the `sci` context _by default_ (i.e. without any additional work required in the scaffolding code that uses `sci` - I'm very well aware that that's possible, in fact I linked to the docs for it in the original issue's "Potential alternatives" section).
